tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py

```diff
@@ -174,7 +174,6 @@
 s2 = RowSchema(
     record_type="M6",
     document=SSP_M6DataSubmissionDocument(),
-    quiet_preparser_errors=True,
```

Comment:
@elipe17 why this change?

Reply:
Looks like this got clobbered in a merge somehow.